package.json

```diff
@@ -175,19 +175,19 @@
     "@types/jest": "^29.5.2",
     "@types/jest-when": "^3.5.2",
     "@types/js-yaml": "^4.0.5",
-    "@types/lodash": "4.14.195",
+    "@types/lodash": "^4.14.195",
     "@types/mock-fs": "^4.13.1",
-    "@types/pusher-js": "5.1.0",
+    "@types/pusher-js": "^5.1.0",
     "@types/react": "^18.2.12",
     "@types/react-collapse": "^5.0.1",
-    "@types/react-dom": "18.2.4",
-    "@types/react-pdf": "5.7.2",
+    "@types/react-dom": "^18.2.4",
+    "@types/react-pdf": "^5.7.2",
     "@types/react-test-renderer": "^18.0.0",
     "@types/semver": "^7.5.0",
     "@types/setimmediate": "^1.0.2",
     "@types/underscore": "^1.11.5",
-    "@typescript-eslint/eslint-plugin": "^5.59.8",
-    "@typescript-eslint/parser": "^5.59.8",
+    "@typescript-eslint/eslint-plugin": "^6.2.1",
+    "@typescript-eslint/parser": "^6.2.1",
```

Comment:
Hey @blazejkustra Looks like a "major" upgrade compared to previous commits, have you tested if everything is working fine with TS?

Reply:
Yes, I double checked every rule that we have for typescript! I also followed the [article](https://typescript-eslint.io/blog/announcing-typescript-eslint-v6) which explains the breaking changes @fabioh8010 

My reasoning behind this change:
- There were some packages conflicting when I installed `5.59.8` version, with `6.2.1` it worked well.
- The newest stable version, with better default rule sets. [recommended-type-checked](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/recommended-type-checked.ts) & [stylistic-type-checked](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/stylistic-type-checked.ts)

Reply:
Just to be sure, please merge it to your POC Onyx branch and test it for yourself that it works as before @fabioh8010 🙌